KarhooUISDK/Screens/AddressScreen/Domain/KarhooRecentAddressProvider.swift

```diff
@@ -22,14 +23,14 @@ final class KarhooRecentAddressProvider: RecentAddressProvider {
         self.persistantStore = persistantStore
     }
 
-    func getRecents() -> [Address] {
+    func getRecents() -> [LocationInfo] {
         return  getExistingAddresses().recents
```

Comment:
We may need to purge the data which is not valid. 
As you can see in the screenshot because I previously had 5 items stored ([Address]) now they are decoded as empty LocationInfo. 

![Simulator Screen Shot - iPhone 12 Pro - 2021-11-16 at 21 19 25](https://user-images.githubusercontent.com/12498117/142068570-a76bb481-e0c0-401f-9c23-ea66e8e396ef.png)

Reply:
Done. To test please: 
1. delete the client app from the device / simulator
2. run the client on the develop branch and select some addresses (i.e. store some soon-to-be invalid data)
3. switch to this branch, run pod install, and run the client again. 
The list should appear empty now, instead of how it's in the screenshot.